code/modules/chemistry/Reagents-ExplosiveFire.dm

```diff
@@ -205,11 +205,8 @@ datum
 				if(!istype(A)) return
 
 				if(holder.get_reagent_amount(id) >= 15) //no more thermiting walls with 1u tyvm
-					var/id = src.id
-					var/datum/reagents/holder = src.holder
-					var/volume
 					holder.del_reagent(id)
-					fireflash_sm(A, 0, rand(20000, 25000) + volume * 2500, 0, 0, 1) // Bypasses the RNG roll to melt walls (Convair880).
+					fireflash_sm(A, 0, rand(20000, 25000), 0, 0, 1) // Bypasses the RNG roll to melt walls (Convair880).
```

Comment:
?

Reply:
Unless I'm really missing something, volume was always 0 (since it's just declared right there) and the other two were just redeclaring the same variables?
Not actually related to the fix anymore, so I can revert if you like.